### PR TITLE
[platform] Got rid of std/*.hpp includes in platform.hpp.

### PR DESCRIPTION
**Pull request by mpimenov**

### CODE REVIEW

**Comment by mpimenov** [comment]
JTTP

**Comment by mpimenov** [comment]
JTTP

File: platform/platform.hpp [hunk 1]
```diff
@@ -89,27 +86,27 @@ class Platform
     Gui
   };
 
-  using TFilesWithType = vector<pair<string, EFileType>>;
+  using TFilesWithType = std::vector<pair<std::string, EFileType>>;
```
Comment (tatiana-yan) [inline platform/platform.hpp]: std::pair

File: platform/platform.hpp [hunk 2]
```diff
@@ -12,13 +12,10 @@
...
-#include "std/function.hpp"
-#include "std/map.hpp"
-#include "std/string.hpp"
-#include "std/unique_ptr.hpp"
-#include "std/utility.hpp"
-#include "std/vector.hpp"
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
```
Comment (tatiana-yan) [inline platform/platform.hpp]: может тогда уж и в platform.cpp? :)

Comment (mpimenov) [reply platform/platform.hpp]: done

File: platform/platform.hpp (hunk 1 above)
Comment (mpimenov) [reply platform/platform.hpp]: done

**Comment by mpimenov** [comment]
JTTP

File: platform/platform.hpp [hunk 3]
```diff
@@ -123,105 +120,108 @@ class Platform
...
-  static bool IsDirectory(string const & path);
+  static bool IsDirectory(std::string const & path);
 
-  static EError GetFileType(string const & path, EFileType & type);
+  static EError GetFileType(std::string const & path, EFileType & type);
 
   /// @return false if file is not exist
   /// @note Check files in Writable dir first, and in ReadDir if not exist in Writable dir
-  bool GetFileSizeByName(string const & fileName, uint64_t & size) const;
+  bool GetFileSizeByName(std::string const & fileName, uint64_t & size) const;
```
Comment (VladiMihaylenko) [inline platform/platform.hpp]: #include <cstdint>

Comment (mpimenov) [reply platform/platform.hpp]: done

**Comment by mpimenov** [comment]
JTTP

**Pull request merged**